go.mod

```diff
@@ -5,20 +5,30 @@ go 1.20
 require (
 	github.com/aws/aws-sdk-go v1.47.9
 	github.com/gin-gonic/gin v1.9.1
+	github.com/maorfr/skbn v0.0.0-20220615055825-326c955b20dd
```

Comment:
Where do we use this package? Thank you, if you check once. 😄

Reply:
I removed all unused packages